sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -867,12 +865,7 @@ def test_online_retrieval_with_go_server(
             )
 
     entity_rows = [
-        {
-            "driver_id": _driver,
-            "customer_id": _customer,
-            "origin_id": origin,
-            "destination_id": destination,
-        }
+        {"origin_id": origin, "destination_id": destination}
```

Comment:
why was this change made?

Reply:
Entities `customer_id` and `driver_id` are not used in this request. And Go implementation explicitly returns error when some unexpected join keys were passed. In comparison Python implementation allow to pass any entities that exists in the registry, but also throw an error when entity name is not recognized.